daemon/api_model_test.go

```diff
@@ -249,28 +250,30 @@ func (s *apiSuite) TestGetModelHasSerialAssertion(c *check.C) {
 	d.overlord.AddManager(deviceMgr)
 	st := d.overlord.State()
 	st.Lock()
+	defer st.Unlock()
 	assertstatetest.AddMany(st, s.storeSigning.StoreAccountKey(""))
 	assertstatetest.AddMany(st, s.brands.AccountsAndKeys("my-brand")...)
 	s.mockModel(c, st, theModel)
 
-	// in case the name of the serial ever changes, just get it state in State
-	// currently it's hard-coded to always be serialserial
-	var authStateData auth.AuthState
-	err = st.Get("auth", &authStateData)
-	c.Assert(err, check.IsNil)
 	serial, err := s.brands.Signing("my-brand").Sign(asserts.SerialType, map[string]interface{}{
 		"authority-id":        "my-brand",
 		"brand-id":            "my-brand",
 		"model":               "my-old-model",
-		"serial":              authStateData.Device.Serial,
+		"serial":              "serialserial",
```

Comment:
a comment on the fact that this is hard-coded somewhere else and that's why we need it to be this would be helpful to future readers I think

Reply:
maybe I should have chosen something else, we are setting it again below here, so it doesn't matter what is hard-coded to somewhere else, the test is self-contained now (at least wrt that)